cmd/integration_link.go

```diff
@@ -447,6 +466,18 @@ func interactiveCreate() (scalingo.SCMRepoLinkCreateParams, error) {
 		hoursBeforeDestroyOnStale := uint(hoursBeforeDestroyOnStale64)
 		params.HoursBeforeDeleteStale = &hoursBeforeDestroyOnStale
 	}
+
+	forksAllowed := false
+	err = survey.AskOne(&survey.Confirm{
+		Message: "Allow review apps to be created from forks:",
+		Help:    "Only allow automatic review apps deployments from forks if you trust the owners of those forks, as this could lead to security issues",
```

Comment:
That help message has been copied from the following US: https://github.com/Scalingo/project-items/issues/77

Reply:
Yep, this has been updated